app/com/linkedin/drelephant/tez/heuristics/MapperMemoryHeuristic.java

```diff
@@ -24,22 +24,25 @@
 
 import org.apache.log4j.Logger;
 
+
 /**
  * Analyzes mapper memory allocation and requirements
  */
 public class MapperMemoryHeuristic extends GenericMemoryHeuristic {
 
   private static final Logger logger = Logger.getLogger(MapperMemoryHeuristic.class);
+  public static final String MAPRED_MAPPER_MEMORY_CONF = "mapreduce.map.memory.mb";
+  public static final String HIVE_MAPPER_MEMORY_CONF = "hive.tez.container.size";
+  public static final String TEZ_MAPPER_MEMORY_CONF = "tez.task.resource.memory.mb";
```

Comment:
I think TEZ_MAPPER_MEMORY_CONF and HIVE_MAPPER_MEMORY_CONF can be defined in GenericMemoryHeuristic so they don't need to be defined in both MapperMemoryHeuristic and MapperMemoryHeuristic.